build-tools/packages/build-tools/src/common/biomeConfigUtils.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Shared utilities for Biome 1.x and 2.x configuration handling.
+ */
+
+import { readFile } from "node:fs/promises";
+import * as JSON5 from "json5";
+
+// switch to regular import once building ESM
```

Comment:
Is this a TODO?

Reply:
Sort of? It's there to explain why that particular import style is used and the conditions under which it can be removed.

Reply:
Nit: Might be worth updating the comment to be a bit more explicit.